README.md

```diff
@@ -69,6 +70,12 @@ This macro converts a string to the data type `nvarchar(2000)` for SQL Server. U
 Usage: 
 `{{ pm_utils.as_varchar('[expression]') }}`
 
+#### create_index ([source](macros/multiple_databases/create_index.sql))
+This macro creates a clustered column index on the input table given as argument for SQL Server.
+
+Usage: 
+`{{ pm_utils.create_index('[input_table]') }}`
+
```

Comment:
- Name of the index is 'clustered columnstore index'.
- Instead of input table it is a 'source table' (to prevent confusion with the tables in the input folder). I would also explicitly mention that the purpose of this macro is to create cci on source tables.
- Document the use of the variable `schema_sources`. We say now in the readme that all possible variables are the format variables. We also do not list it in the docs of this macro.

Reply:
For the first bullet, you mean that we should rename the macro to `create_clustered_columnstore_index`? Maybe `create_cc_index`?